scRNA-seq/.gitignore

```diff
@@ -3,34 +3,12 @@ index
 
 gene-sets
 
-data/glioblastoma/preprocessed
-data/hodgkins/cellranger
-
-data/mouse-liver
-
-data/PBMC-TotalSeqB
-
-data/tabula-muris/fastq
-data/tabula-muris/fastq-raw
-data/tabula-muris/normalized/TM_normalized.rds
-
-data/glioblastoma-old
-
-data/reference
-
+# ignore data directory, except for some places to keep
+data/
+!data/reference/.gitkeep
+!data/tabula-muris/normalized/.gitkeep
```

Comment:
I was assuming either keep the files or nothing? I'm not sure the function of keeping the directories here. They should be created either by the data linking script or S3 sync. (In my earlier comment I didn't think about the fact that the reference data is all linked from the shared directory.)